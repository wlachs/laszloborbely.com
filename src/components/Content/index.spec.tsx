import { cleanup, render, screen } from '@testing-library/react';
import { afterEach, describe, expect, it } from 'vitest';

import { Content } from './';

describe('Content component', () => {
	afterEach(() => {
		cleanup();
	});

	it('should always render child elements', () => {
		const child = 'child';
		const { container } = render(<Content>{child}</Content>);
		expect(screen.getByText(child)).toBeDefined();
		expect(container).toMatchSnapshot();
	});
});
