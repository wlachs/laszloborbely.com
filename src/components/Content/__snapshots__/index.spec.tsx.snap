// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Content component > should always render child elements 1`] = `
<div>
  <div
    class="flex w-full flex-grow flex-col items-center"
  >
    child
  </div>
</div>
`;
